File: 3/programming/week6/30-22126-nishiyama_sou-kadai-6.c
#include <stdio.h>

double max(double a, double b, double c){
	if (a>=b && a>=c){
		return a;
	}
	else if (b>=a && b>=c){
		return b;
	}
	else if (c>=a && c>=b){
		return c;
	}
}

double min(double a, double b, double c){
	if (a<=b && a<=c){
		return a;
	}
	else if (b<=a && b<=c){
		return b;
	}
	else if (c<=a && c<=b){
		return c;
	}
}

double sum (double a, double b, double c){
	return a+b+c;	
}

double avg (double a, double b, double c){
	return sum(a,b,c)/3;
}

int main(void){
	double a,b,c;
	int flg;
	scanf("%lf %lf %lf", &a, &b, &c);
	printf("max: 1, min: 2, sum: 3, avg: 4 \n");
	scanf("%d", &flg);
	if (flg == 1){
		printf("%lf", max(a,b,c));
	} 
	else if (flg == 2){
		printf("%lf", min(a,b,c));
	} 
	else if (flg == 3){
		printf("%lf", sum(a,b,c));
	}
	else if (flg == 4){
		printf("%lf", avg(a,b,c));
	}
	//printf("max %lf, min %lf, sum %lf, avg %lf", max(a,b,c), min(a,b,c), sum(a,b,c), avg(a,b,c));
	printf("\n");
	return 0;
}

File: 3/programming/week6/30-22126-nishiyama_sou-kadai-6.py
def ownMax(a,b,c):
	if a>=b and a>=c:
		return a
	elif b>=c and b>=a:
		return b
	elif c>=a and c>=b:
		return c

def ownMin(a,b,c):
	if a<=b and a<=c:
		return a
	elif b<=c and c<=a:
		return b
	elif c<=a and c<=b:
		return c


def ownSum(a,b,c):
	return a+b+c

def ownAvg(a,b,c):
	return ownSum(a,b,c)/3


a = float(input())
b = float(input())
c = float(input())
print("max: 1, min: 2, sum: 3, avg: 4")
flg = int(input())

if flg == 1:
	print(ownMax(a,b,c))
elif flg == 2:
	print(ownMin(a,b,c))
elif flg == 3:
	print(ownSum(a,b,c))
elif flg == 4:
	print(ownAvg(a,b,c))
